### PR TITLE
{tools}[intel/2019b] NCO v5.0.6

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3519.doduo.os - Linux RHEL 8.4, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/0ae7dbf191b63dadb133b415e6ffcc57 for a full test report.

File: easybuild/easyconfigs/n/NCO/NCO-5.0.6-intel-2019b.eb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+name = 'NCO'
+version = '5.0.6'
+
+homepage = "https://nco.sourceforge.net"
+description = """manipulates and analyzes data stored in netCDF-accessible formats, including DAP, HDF4, and HDF5"""
+
+toolchain = {'name': 'intel', 'version': '2019b'}
+# compile with -O1 to work around Intel compiler crash (Segmentation violation signal raised)
+toolchainopts = {'lowopt': True}
```
[the remark this answers is not in the document]
Comment (smoors) [reply easybuild/easyconfigs/n/NCO/NCO-5.0.6-intel-2019b.eb:11]: @boegel can you add this useful info as a comment (even if it's not complete)?
I just tested this in Broadwell, which works fine too.

File: easybuild/easyconfigs/n/NCO/NCO-5.0.6-intel-2019b.eb [hunk 2]
```diff
@@ -0,0 +1,46 @@
+easyblock = 'ConfigureMake'
+
+name = 'NCO'
+version = '5.0.6'
+
+homepage = "https://nco.sourceforge.net"
+description = """manipulates and analyzes data stored in netCDF-accessible formats, including DAP, HDF4, and HDF5"""
+
+toolchain = {'name': 'intel', 'version': '2019b'}
+# compile with -O1 to work around Intel compiler crash (Segmentation violation signal raised)
```
Comment (branfosj) [inline easybuild/easyconfigs/n/NCO/NCO-5.0.6-intel-2019b.eb:10]:
```suggestion
# compile with -O1 to work around Intel compiler crash (Segmentation violation signal raised)
# which happens on some processors (such as skylake, cascadelake)
```

File: easybuild/easyconfigs/n/NCO/NCO-5.0.6-intel-2019b.eb (hunk 1 above)
[the remark this answers is not in the document]
Comment (branfosj) [reply easybuild/easyconfigs/n/NCO/NCO-5.0.6-intel-2019b.eb:12]: done in 7efdc28

**Review by branfosj — APPROVED** [review]

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 7 out of 7 (1 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/branfosj/79cbe9b053526c8f736b6adf4eb0be15 for a full test report.

**Comment by branfosj** [comment]
Going in, thanks @boegel!

**Pull request merged**